### PR TITLE
[DOC beta] Update Ember.Component code samples to avoid globals

### PR DESCRIPTION
**Pull request by greyhwndz**

### CODE REVIEW

File: packages/ember-views/lib/views/component.js [hunk 1]
```diff
@@ -36,16 +36,17 @@ function validateAction(component, actionName) {
...
   The easiest way to create an `Ember.Component` is via
   a template. If you name a template
-  `components/my-foo`, you will be able to use
-  `{{my-foo}}` in other templates, which will make
+  `components/app-profile`, you will be able to use
+  `{{app-profile}}` in other templates, which will make
   an instance of the isolated component.
 
   ```handlebars
+  <!-- app/templates/application.hbs -->
```
Comment (rwjblue) [inline packages/ember-views/lib/views/component.js]: We should use handlebars comments here instead. 

{{! Comment here }}

**Comment by rwjblue** [comment]
Thanks for taking the time to submit a PR!

Can you update the various comments in templates here to use handlebars style comments?

Also, it looks like this has merge conflicts so you may need to rebase.

**Comment by greyhwndz** [comment]
@rwjblue: Done. Would be happy to wait for feedback about how this could further be improved. Thanks.

**Comment by greyhwndz** [comment]
@rwjblue: changed a missed handlebars comment

**Comment by greyhwndz** [comment]
Closing this because of some repo prob... opened another PR instead...

**Pull request closed without merging**